### PR TITLE
docs: include `--acknowledge-secrets-access` flag description

### PR DESCRIPTION
**Pull request by paragbhingre**
Using `--acknowledge-secrets-access` flag secrets access permission grant question will be skipped.

### CODE REVIEW

File: site/content/docs/commands/task-exec.en.md [hunk 1]
```diff
@@ -8,14 +8,16 @@ $ copilot task exec
...
-      --task-id string   Optional. ID of the task you want to exec in.
+  -a, --app string                  Name of the application.
+  -c, --command string              Optional. The command that is passed to a running container. (default "/bin/bash")
+      --default                     Optional. Execute commands in running tasks in default cluster and default subnets.
+                                    Cannot be specified with 'app' or 'env'.
+  -e, --env string                  Name of the environment.
+  -h, --help                        help for exec
+  -n, --name string                 Name of the service, job, or task group.
+      --task-id string              Optional. ID of the task you want to exec in.
+      --acknowledge-secrets-access  Optional. Skips the secrets access permission grant question and directly grants the
```
Comment (iamhopaul123) [inline site/content/docs/commands/task-exec.en.md]: you sure it is for task exec?

Comment (paragbhingre) [reply site/content/docs/commands/task-exec.en.md]: Ooo, I was making changes in the task-run and by mistake pasted code in task-exec. I will change it.

File: site/content/docs/commands/task-run.en.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ Generally, the steps involved in task run are:
     --tag string                     Optional. The container image tag in addition to "latest".
 -n, --task-group-name string         Optional. The group name of the task. Tasks with the same group name share the same set of resources.
     --task-role string               Optional. The role for the task to use.
+     --acknowledge-secrets-access    Optional. Skips the secrets access permission grant question and directly grants the permission to secrets.
```
Comment (Lou1415926) [inline site/content/docs/commands/task-run.en.md]: We should copy our actual help menu here 😛 It says "Optional. Skip the confirmation question and grant access to the secrets specified by --secrets flag. 
This flag is useful only when '--secret' flag is specified"

**Review by iamhopaul123 — APPROVED** [review]

**Review by Lou1415926 — APPROVED** [review]

File: site/content/docs/commands/task-run.en.md [hunk 3]
```diff
@@ -56,6 +56,8 @@ Generally, the steps involved in task run are:
     --tag string                     Optional. The container image tag in addition to "latest".
 -n, --task-group-name string         Optional. The group name of the task. Tasks with the same group name share the same set of resources.
     --task-role string               Optional. The role for the task to use.
+     --acknowledge-secrets-access    Optional. Skip the confirmation question and grant access to the secrets specified by --secrets flag.
```
Comment (dannyrandall) [inline site/content/docs/commands/task-run.en.md]:
```suggestion
    --acknowledge-secrets-access    Optional. Skip the confirmation question and grant access to the secrets specified by --secrets flag.
```
nit

File: site/content/docs/commands/task-run.en.md [hunk 4]
```diff
@@ -56,6 +56,8 @@ Generally, the steps involved in task run are:
     --tag string                     Optional. The container image tag in addition to "latest".
 -n, --task-group-name string         Optional. The group name of the task. Tasks with the same group name share the same set of resources.
     --task-role string               Optional. The role for the task to use.
+    --acknowledge-secrets-access    Optional. Skip the confirmation question and grant access to the secrets specified by --secrets flag.
```
Comment (dannyrandall) [inline site/content/docs/commands/task-run.en.md]: lol. sorry, this is my bad 🙃 
```suggestion
    --acknowledge-secrets-access     Optional. Skip the confirmation question and grant access to the secrets specified by --secrets flag.
```

**Pull request merged**